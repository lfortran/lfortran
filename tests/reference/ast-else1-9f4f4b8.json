{
    "basename": "ast-else1-9f4f4b8",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form/else1.f",
    "infile_hash": "90793a3b11cf9eb1bd0bc949e1ed82c1559710f30707cc69106dfdcd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-else1-9f4f4b8.stdout",
    "stdout_hash": "ff1564bd723cd1e31db5e3e5663ebe9f10c9e32cce684748c666100d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}