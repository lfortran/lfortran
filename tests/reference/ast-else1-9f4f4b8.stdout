(TranslationUnit
    [(Subroutine
        fun1
        []
        []
        ()
        ()
        []
        []
        []
        []
        [(If
            0
            ()
            (> (FuncCallOrArray
                abs
                []
                [(()
                bjv0
                ()
                0)]
                []
                []
            ) (FuncCallOrArray
                abs
                []
                [(()
                bjv1
                ()
                0)]
                []
                []
            ))
            [(=
                0
                cs
                (/ bjv0 f)
                ()
            )]
            []
            ()
            ()
            ()
        )
        (=
            0
            elsecs
            (/ bjv1 f2)
            ()
        )]
        []
    )
    (Subroutine
        fun2
        []
        []
        ()
        ()
        []
        []
        []
        []
        [(If
            0
            ()
            (Logical .true.)
            [(=
                0
                x
                1
                ()
            )
            (If
                0
                ()
                (> (FuncCallOrArray
                    abs
                    []
                    [(()
                    bjv0
                    ()
                    0)]
                    []
                    []
                ) (FuncCallOrArray
                    abs
                    []
                    [(()
                    bjv1
                    ()
                    0)]
                    []
                    []
                ))
                [(=
                    0
                    cs
                    (/ bjv0 f)
                    ()
                )]
                []
                ()
                ()
                ()
            )
            (=
                0
                elsecs
                (/ bjv1 f2)
                ()
            )]
            []
            ()
            ()
            ()
        )]
        []
    )
    (Subroutine
        fun3
        []
        []
        ()
        ()
        []
        []
        []
        []
        [(If
            0
            ()
            (Logical .true.)
            [(=
                0
                x
                1
                ()
            )
            (If
                0
                ()
                (> (FuncCallOrArray
                    abs
                    []
                    [(()
                    bjv0
                    ()
                    0)]
                    []
                    []
                ) (FuncCallOrArray
                    abs
                    []
                    [(()
                    bjv1
                    ()
                    0)]
                    []
                    []
                ))
                [(=
                    0
                    cs
                    (/ bjv0 f)
                    ()
                )]
                [(=
                    0
                    cs
                    (/ bjv1 f2)
                    ()
                )]
                ()
                ()
                ()
            )]
            []
            ()
            ()
            ()
        )]
        []
    )]
)
